flow/src/test/scala/org/alephium/flow/core/FlowUtilsSpec.scala

```diff
@@ -98,6 +98,25 @@ class FlowUtilsSpec extends AlephiumSpec {
     }
     val newTx    = Transaction.from(tx.unsigned.inputs, outputs, tx.inputSignatures)
     val newBlock = block.copy(transactions = AVector(newTx))
+
+    val ts0 = System.currentTimeMillis()
     blockFlow.add(newBlock).isRight is true
+    val ts1 = System.currentTimeMillis()
+
+    val (balance, lockedBalance, utxos) = blockFlow.getBalance(output.lockupScript).rightValue
+    balance is U256.unsafe(outputs.sumBy(_.amount.toBigInt))
+    lockedBalance is 0
+    utxos is 10000
+
+    val ts2 = System.currentTimeMillis()
+    blockFlow.prepareUnsignedTx(
+      keyManager(output.lockupScript).publicKey,
+      output.lockupScript,
+      None,
+      ALF.oneAlf
+    )
+    val ts3 = System.currentTimeMillis()
+
+    print(s"Times: ${ts1 - ts0} - ${ts2 - ts1} - ${ts3 - ts2}\n")
```

Comment:
do you really want to keet that or it was only for dev purpose?

Reply:
Yep, just want to see the performance for large utxo set